mis_builder_analytic_filter/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+MIS Builder Analytic Filter
+===========================
+
+This module extends the functionality of mis_builder to
+provide filtering on analytic axis.
+
+Installation
+============
+
+There is no specific installation procedure for this module.
+
+Configuration and Usage
+=======================
+
+In Accounting > Reporting > MIS Reports, there is a new field
+'Analytic Account' which can be used to filter move line on an
+analytic axis and it's children.
```

Comment:
It's about the children of the analytic axis (singular) so I'd keep "its" (without the apostrophe). And I add a "s" to move line*s*.